src/LevelLoader.cpp

```diff
@@ -296,3 +296,19 @@ ActionComponent* LevelLoader::getActionComponent(json obj) {
 
    return nullptr;
 }
+
+//TODO(nurgan) find a better method for parsing to enum
+GameObjectType LevelLoader::stringToType(std::string type) {
```

Comment:
Honestly GameObject should probably just have a static function that does this conversion instead of in here. Then we keep this isolated with the GameObject code and not clustered in the LevelLoader. Other options could be to make a map<String, Enum> that is statically defined in the GameObject that has all of that data but I don't know if that is more or less elegant, lookup would be slightly faster for sure though. @rgarmsen2295 what do you think?